File: Functions/hello_world_update.py
# Jameson Bultez, First Program Python

# Takes in a name and returns string "Hello, (name)."
def hey_you(name):
    return f"Hello, {name}."

# Asks for user's first name.
user_name = input("What be thy first name?:")

# Prints "Hello, (user_name)."
print(hey_you(user_name))

# Introduces other print statments.
print("These are some other people:")

# Prints additional hello statements.
print(hey_you("Jacob"))
print(hey_you("Sarah"))
print(hey_you("Scott"))
print(hey_you("Hannah"))

File: Functions/helloWorldUpdate.c
// Jameson Bultez, Hello World Update C

#include <stdio.h>

// Declare variable that will be defined as the user's first name.
char user_name[50];

// Function to print "Hello, (name)."
void helloWorld(char name[50]) {
    printf("Hello, %s.\n", name);
}

int main(void) {

    // Get user's first name and store it in user_name.
    printf("Type your first name: ");
    scanf("%s", user_name);

    // Print hello to user and various other names.
    helloWorld(user_name);
    helloWorld("Marcus");
    helloWorld("Mia");
    helloWorld("Miles");
    helloWorld("Maria");

    return 0;
}